pkg/mcs/tso/server/grpc_service.go

```diff
@@ -164,7 +163,7 @@ func (s *Service) FindGroupByKeyspaceID(
 			Address: member.Address,
 			// TODO: watch the keyspace groups' primary serving address changes
 			// to get the latest primary serving addresses of all keyspace groups.
-			IsPrimary: strings.EqualFold(member.Address, am.GetLeaderAddr()),
+			IsPrimary: member.CompareAddress(am.GetLeaderAddr()),
```

Comment:
Do we need to keep consistent with cse?

Reply:
No, pd-cse will be switched to release-7.5